### ISSUE TITLE
Chat spammed with this error

### ISSUE THREAD

**Issue by u9g** [PrismarineJS/mineflayer#1041]
## Versions
 - mineflayer: newest
 - server: spigot
 - node: newest

## Detailed description of a problem
Logs are spammed with this error for seemingly no reason, and doesn't affect bot at all? It never happened in any previous mineflayer versions

```
Error: Attempted to read beyond the bounds of the managed data.
    at SmartBuffer.ensureReadable (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\smart-buffer\build\smartbuffer.js:1093:19)
    at SmartBuffer._readNumberValue (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\smart-buffer\build\smartbuffer.js:1163:14)
    at SmartBuffer.readInt32BE (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\smart-buffer\build\smartbuffer.js:136:21)
    at ChunkColumn.load (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\prismarine-chunk\src\pc\1.13\ChunkColumn.js:239:35)
    at addColumn (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\mineflayer\lib\plugins\blocks.js:93:14)
    at Client.<anonymous> (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\mineflayer\lib\plugins\blocks.js:295:5)
    at Client.emit (events.js:310:20)
    at FullPacketParser.<anonymous> (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\minecraft-protocol\src\client.js:87:12)
    at FullPacketParser.emit (events.js:310:20)
    at addChunk (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\readable-stream\lib\_stream_readable.js:298:12)
    at readableAddChunk (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\readable-stream\lib\_stream_readable.js:280:11)
    at FullPacketParser.Readable.push (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\readable-stream\lib\_stream_readable.js:241:10)
    at FullPacketParser.Transform.push (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\readable-stream\lib\_stream_transform.js:139:32)
    at FullPacketParser._transform (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\protodef\src\serializer.js:81:10)
    at FullPacketParser.Transform._read (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\readable-stream\lib\_stream_transform.js:177:10)
    at FullPacketParser.Transform._write (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\readable-stream\lib\_stream_transform.js:164:83)
    at doWrite (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\readable-stream\lib\_stream_writable.js:409:139)
    at writeOrBuffer (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\readable-stream\lib\_stream_writable.js:398:5)
    at FullPacketParser.Writable.write (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\readable-stream\lib\_stream_writable.js:307:11)
    at Decompressor.ondata (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\readable-stream\lib\_stream_readable.js:681:20)
    at Decompressor.emit (events.js:310:20)
    at addChunk (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\readable-stream\lib\_stream_readable.js:298:12)
    at readableAddChunk (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\readable-stream\lib\_stream_readable.js:280:11)
    at Decompressor.Readable.push (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\readable-stream\lib\_stream_readable.js:241:10)
    at Decompressor.Transform.push (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\readable-stream\lib\_stream_transform.js:139:32)
    at Unzip.cb (C:\Users\---\Documents\js-projects\new-cosmic-bot\node_modules\minecraft-protocol\src\transforms\compression.js:69:14)
    at Unzip.zlibBufferOnEnd (zlib.js:149:10)
    at Unzip.emit (events.js:322:22)
    at endReadableNT (_stream_readable.js:1187:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
```

## Expected behavior
just don't show it

**Comment by Karang** [comment]
Hiding an error is not a solution, the cause should be fixed...
Looking at:
https://github.com/PrismarineJS/prismarine-chunk/blob/master/src/pc/1.13/ChunkColumn.js#L239
it seems the biome data are not sent by the server.
According to https://wiki.vg/Chunk_Format#Packet_structure `Biomes` is an `Optional array of Integer | 1024 biome IDs, ordered by x then z then y, in 4×4×4 blocks.  Not present if full chunk is false.`
It's likely related to this issue https://github.com/PrismarineJS/prismarine-chunk/issues/43

**Comment by u9g** [comment]
@Karang so what am I to do then?

**Comment by Karang** [comment]
Either fix https://github.com/PrismarineJS/prismarine-chunk/issues/43 or wait for someone to fix it

**Comment by rom1504** [comment]
fixed

**Comment by Karang** [comment]
Almost, someone needs to change https://github.com/PrismarineJS/mineflayer/blob/master/lib/plugins/blocks.js#L93
to:
```js
column.load(args.data, args.bitMap, args.skyLightSent, args.groundUp)
```

**Comment by rom1504** [comment]
ah right, forgot this

**Comment by u9g** [comment]
still getting this problem

**Comment by Karang** [comment]
still waiting for someone to fix it

**Comment by u9g** [comment]
ah i thought above rom fixed it

**Comment by rom1504** [comment]
Fixed and released as 2.19.2
